src/js/components/DeploymentsListComponent.jsx

```diff
@@ -107,6 +168,11 @@ var DeploymentListComponent = React.createClass({
               Error fetching deployments. Refresh to try again.
             </td>
           </tr>
+          <tr className={errorMessageClassSet}>
+            <td className="text-center text-danger" colSpan="5">
+              {state.errorMessage}
+            </td>
+          </tr>
```

Comment:
Hmm, no it doesn't. :) Apps should have it above, like in deployments. Else on an existing apps list, which is very long, this message wouldn't be visible.
So, could you create an issue for that to bring the error message in the apps view above "No running apps"?